treeherder/etl/perf.py

```diff
@@ -100,68 +100,42 @@ def load_perf_artifacts(project_name, reference_data, job_data, datum):
 
     framework = PerformanceFramework.objects.get(name=perf_datum['framework']['name'])
     for suite in perf_datum['suites']:
-        subtest_signatures = []
+        subtest_properties = []
+        summary_signature_hash = None
         for subtest in suite['subtests']:
-            subtest_properties = {
+            subtest_propertie = {
                 'suite': suite['name'],
-                'test': subtest['name']
+                'test': subtest['name'],
+                'lowerIsBetter': subtest.get('lowerIsBetter', True),
+                'value': subtest['value']
```

Comment:
You're updating the signature here based on *all* the subtest properties, which won't work for calculating the signature. In particular, every time "value" changes (which is every time we run a performance test, basically), you'll have a new performance signature!

You need to at least take out "value" here.